docs/windows-client.md

```diff
@@ -4,36 +4,21 @@
 
 This documentation has the goal of showing how a user can use the official Windows [Tailscale](https://tailscale.com) client with `headscale`.
 
-## Add registry keys
-
-To make the Windows client behave as expected and to run well with `headscale`, two registry keys **must** be set:
+## Installation
 
-- `HKLM:\SOFTWARE\Tailscale IPN\UnattendedMode` must be set to `always` as a `string` type, to allow Tailscale to run properly in the background
-- `HKLM:\SOFTWARE\Tailscale IPN\LoginURL` must be set to `<YOUR HEADSCALE URL>` as a `string` type, to ensure Tailscale contacts the correct control server.
+Download the [Official Windows Client](https://tailscale.com/download/windows) and install it.
 
-You can set these using the Windows Registry Editor:
+When the installation has finished, close the tailscale application in the windows tray. (Richt click > Exit)
 
-![windows-registry](./images/windows-registry.png)
```

Comment:
@kradalby suggested in https://github.com/juanfont/headscale/pull/2096#issuecomment-2325761456 to keep the registry instructions but probably degrade their need from "required" to "optional" and move into their own section.

If you remove them, you should also delete the screenshot: `./docs/images/windows-registry.png`

Reply:
Yes, based on the PR, it is implied that they dont work anymore, I dont think they have been removed, so not sure why that is.

Reply:
I quickly looked into this and it seems that Tailscale changed the location of registries from `HKLM:\SOFTWARE\Tailscale IPN` to `HKLM\Software\Policies` around 2022, so the paths and the generated reg file are wrong. See https://github.com/tailscale/tailscale/issues/3584#issuecomment-1034277652 and Tailscale's  [Deploy Tailscale on Windows using MDM](https://tailscale.com/kb/1318/windows-mdm) documentation.

Since our minimal client versions have fast user switching, we should follow Tailscale's recommendation to login: `tailscale login --login-server=<url>`.

The remaining issue is "unattended mode". It seemed to be a mess in the past, but now:

* the user can enable unattended mode in the GUI ("Preferences" &rarr; "Run unattended").
* since end of 2022, this preference seems to be synced to the Windows service - see: https://github.com/tailscale/tailscale/pull/6255

----
The simplest way to solve this seems to be:

* Rewrite `/windows` to basically: 
  * include a link to the Tailscale Windows download page
  * Print the login command
  * Minimal setup instructions
* Cleanup the windows documentation accordingly and add a section about "unattended mode" which should advise the user to enable it via GUI. Probably also add the link to [Deploy Tailscale on Windows using MDM](https://tailscale.com/kb/1318/windows-mdm) as it contains the details about the registry keys.

Reply:
Alternatively, it might be a good time to remove `/apple` and `/windows` altogether? They provide little value with the newer Tailcale clients and there is also headscale's documentation at https://headscale.net which can be updated faster.

Reply:
I'm hesitant at removing them, they feel useful, but can be as simple as possible. I find that it is slightly neater than the docs because the command can be generated with the correct url and copied as if.

Otherwise I agree with your other suggestions.